site2/docs/concepts-architecture-overview.md

```diff
@@ -139,23 +139,3 @@ Some important things to know about the Pulsar proxy:
 * Connecting clients don't need to provide *any* specific configuration to use the Pulsar proxy. You won't need to update the client configuration for existing applications beyond updating the IP used for the service URL (for example if you're running a load balancer over the Pulsar proxy).
 * [TLS encryption](security-tls-transport.md) and [authentication](security-tls-authentication.md) is supported by the Pulsar proxy
 
-## Service discovery
```

Comment:
This section is describing a bit how the "pulsar service discovery" mechanism works and not about the deleted "DiscoveryService" component

Reply:
Thx, I see 👍